src/Common/src/System/Net/WebSockets/ManagedWebSocket.common.cs

```diff
@@ -26,22 +25,8 @@ namespace System.Net.WebSockets
     ///   a send operation while another is in progress or a receive operation while another is in progress will
     ///   result in an exception.
     /// </remarks>
-    internal sealed class ManagedWebSocket : WebSocket
+    internal sealed partial class ManagedWebSocket : WebSocket
     {
-        /// <summary>Creates a <see cref="ManagedWebSocket"/> from a <see cref="Stream"/> connected to a websocket endpoint.</summary>
-        /// <param name="stream">The connected Stream.</param>
-        /// <param name="isServer">true if this is the server-side of the connection; false if this is the client-side of the connection.</param>
-        /// <param name="subprotocol">The agreed upon subprotocol for the connection.</param>
-        /// <param name="keepAliveInterval">The interval to use for keep-alive pings.</param>
-        /// <param name="receiveBufferSize">The buffer size to use for received data.</param>
-        /// <param name="receiveBuffer">Optional buffer to use for receives.</param>
-        /// <returns>The created <see cref="ManagedWebSocket"/> instance.</returns>
-        public static ManagedWebSocket CreateFromConnectedStream(
-            Stream stream, bool isServer, string subprotocol, TimeSpan keepAliveInterval, Memory<byte> buffer)
```

Comment:
I understand why you've made the split you've made, but there's a ton of duplication here.  I don't think it's necessary, and it's going to make it harder to evolve the implementation.

I would prefer to see as minimal changes made as possible, and I don't think a ton should be needed.  For example:
- The new overrides can be moved to a separate .netcoreapp.cs file.
- The netstandard implementation can reference System.Memory so that it can use `Memory<T>`, `Span<T>`, etc.
- Very few new APIs are actually used; in places where they are, you can abstract those away with extension methods that on netcoreapp just target the new methods and on netstandard have a polyfill implementation, e.g. Task.IsCompletedSuccessfully can just do Task.Status == TaskStatus.RanToCompletion, Stream.WriteAsync(Memory) can extra the array and call Stream.WriteAsync(byte[], int, int), etc.